projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -593,7 +593,10 @@ public function get_all_integrations_status() {
 		foreach ( array_keys( $this->get_supported_integrations() ) as $slug ) {
 			// For now, we only have plugin integrations.
 			// When needed, handle other integration types here.
-			$integrations[ $slug ] = $this->get_plugin_status( $slug );
```

Comment:
Since the data is changing should this be different version of the endpoint?

Reply:
I thought about it, and in theory yes, but I'm not sure it's needed or worth it here. This is a very new endpoint and used internally, by us, in only one specific spot (the useIntegratoinsStatus hook). 

Also, the way endpoints are defined in this part of the code, we have not be versioning any of them. The full endpoint with the rest_base is `/wp/v2/feedback/integrations`. It's not obvious how we adjust this for versioning.

So I'd say we skip, but I'd also be fine introducing versioning for it if others think it's warranted.

Reply:
Updated. Endpoint takes in a version param, and if not set or set to 1, it will return the old data format. If set to 2, it will return the new data format.